### DISCUSSION TITLE
Ability to access custom (non-project) boards

### DISCUSSION THREAD

**Discussion by siwilkins** [ankitpokhrel/jira-cli#418] [Ideas] [answered]
Hi,

Am really enjoying the CLI, however I have a Jira board which gives me oversight of all issues assigned to me, regardless of project. As such, it doesn't belong to a project itself. It doesn't seem, from what I can see, that there is an ability to access this board, as the board option is always scoped to a project (either my default or passed in as a parameter.

Thanks very much!

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @siwilkins,

The command below should give you all issues assigned to you in all projects.
```sh
jira issue list -a$(jira me) -q "project IS NOT EMPTY"
```

**Comment by siwilkins** [comment]
Fab - thanks very much!
